documentation/common/address-space.adoc

```diff
@@ -127,3 +127,52 @@ The multicast address type is a scalable direct address for sending messages to
 * standard
 
 // address.multicast.plan.inmemory.shortDescription:Configures router network with multicast address
+
+[[brokered_address_space]]
+=== Brokered Address Space
```

Comment:
One question is whether it would be worth keeping the address spaces in distinct module files(?).